app/assets/stylesheets/govuk_publishing_components/components/_tabs.scss

```diff
@@ -7,3 +7,19 @@
     border: 0;
   }
 }
+
+// We have some styles within GOVUK (.content-block) which can leak into the list styles for this component.
```

Comment:
I'd suggest linking or at least referencing exactly where that code is as you have in the PR description, so people can potentially clean up later with a quick check.